### PR TITLE
New Extension: HackMD

### PR DESCRIPTION
**Pull request by Yukaii**
## Description

Extension for [HackMD](https://hackmd.io)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/hackmd/package.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    "Applications",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "browse-notes",
+      "title": "Browse Notes",
+      "subtitle": "HackMD",
+      "description": "Browser HackMD Notes",
```
Comment (danielsequeira) [inline extensions/hackmd/package.json:20]:
```suggestion
      "description": "Browse HackMD Notes",
```

File: extensions/hackmd/package.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      "description": "The API token you created on HackMD",
+      "name": "api_token",
+      "required": true
+    },
+    {
+      "type": "textfield",
+      "title": "HackMD API Base URL",
+      "description": "The base URL of HackMD API",
+      "name": "api_base_url",
+      "default": "https://api.hackmd.io/v1",
```
Comment (danielsequeira) [inline extensions/hackmd/package.json:58]: isn't this always the same or it varies for every HackMD user/account?

File: extensions/hackmd/package.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      "required": false
+    },
+    {
+      "type": "textfield",
+      "title": "HackMD Instance URL",
+      "description": "The base URL of HackMD instance",
+      "name": "instance_url",
+      "default": "https://hackmd.io",
```
Comment (danielsequeira) [inline extensions/hackmd/package.json:66]: isn't this always the same or it varies for every HackMD user/account?

File: extensions/hackmd/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# HackMD Changelog
+
+## [Initial Version] - 2022-07-29
```
Comment (danielsequeira) [inline extensions/hackmd/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - 2022-08-04
```

**Comment by danielsequeira** [comment]
Hi @Yukaii 👋 

Thanks for your contribution.

I just went through the code and left a few small comments.

Tomorrow I will test the extension as well.

File: extensions/hackmd/package.json (hunk 2 above)
Comment (Yukaii) [reply extensions/hackmd/package.json:58]: HackMD has an enterprise edition for self-hosting instances, so I leave an option here.

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**